tensilelite/Tensile/Source/lib/source/ContractionSolution.cpp

```diff
@@ -986,13 +990,35 @@ namespace Tensile
             internalArg0      = internalArg0 | wgShift8;
         }
 
+        if(internalArgsSupport.wgm && internalArgsSupport.version >= 1)
+        {
+            if(internalArgsSupport.version == 1)
+            {
+                internalArg1 = wgm;
+            }
+            else if(internalArgsSupport.version == 2)
+            {
+                wgmxcc = param.wgmxcc() > 1 ? param.wgmxcc() : sizeMapping.workGroupMappingXCC;
```

Comment:
default value of param.wgmxcc() should be 0. wgmxcc should use the value of param.wgmxcc() if param.wgmxcc() > 0.

Reply:
updated